src/processor/operator/persistent/node_batch_insert.cpp

```diff
@@ -151,11 +158,14 @@ void NodeBatchInsert::writeAndResetNodeGroup(transaction::Transaction* transacti
     std::unique_ptr<ChunkedNodeGroup>& nodeGroup, std::optional<IndexBuilder>& indexBuilder) const {
     const auto nodeSharedState =
         ku_dynamic_cast<BatchInsertSharedState*, NodeBatchInsertSharedState*>(sharedState.get());
+    const auto nodeLocalState =
+        ku_dynamic_cast<BatchInsertLocalState*, NodeBatchInsertLocalState*>(localState.get());
```

Comment:
Use `ptrCast` instead of `ku_dynamic_cast` here.

Reply:
updated